ui/src/main/webapp/css/windup-web.css

```diff
@@ -112,9 +112,9 @@ td {
     font-size: 12px;
 }
 
-a[target='_blank']::before {
+.external-link::before {
```

Comment:
I'm not sure if external link is better than `a[target='_blank']` - I think it is easier to forget to add that class. But we might use both of them, just to be sure?

I guess you didn't like `a[target='_blank']` being too global, right? Are we ever going to use it and require it not to have that icon? (Maybe having it with icon as default would be better than opposite? So in case we don't want to have an icon, we would have to use more specific selector and style it not to include icon).